content/developer/reference/frontend/services.rst

```diff
@@ -455,6 +457,100 @@ A notification that closes after a second:
   const close = notificationService.add("I will be quickly closed");
   setTimeout(close, 1000);
 
+.. _frontend/services/router:
+
+Router Service
+--------------
+
+Overview
+~~~~~~~~
+
+- Technical name: `router`
+- Dependencies: none
+
+The `router` service provides three features:
+
+* information about the current route
+* provides a way for the application to update the url, depending on its state
+* listens to every hash change, and notifies the rest of the application
```

Comment:
maybe change the sentence not to start with a verb to be consistent with the two other points ?